WebDriverAgentLib/Utilities/FBCapabilities.h

```diff
@@ -11,8 +11,6 @@
 
 /** Whether to use alternative elements visivility detection method */
 extern NSString* const FB_CAP_USE_TEST_MANAGER_FOR_VISIBLITY_DETECTION;
-/** Set the maximum amount of characters that could be typed within a minute (60 by default) */
-extern NSString* const FB_CAP_MAX_TYPING_FREQUENCY;
```

Comment:
It still makes sense to keep the constant, so it would be obvious it's present in both capabilities and settings

Reply:
Thank you. Fixed in https://github.com/appium/WebDriverAgent/pull/904/commits/75d26ca34ccdee7fdfbb5318d346cbf0a9a822b0